checks.d/mysql.py

```diff
@@ -863,12 +863,17 @@ def _get_replica_stats(self, db):
             self.warning("Privileges error getting replication status (must grant REPLICATION CLIENT): %s" % str(e))
             return {}
 
-    def _get_slave_status(self, db, nonblocking=True):
+    def _get_slave_status(self, db, above_560):
```

Comment:
I'm not sure if that was added on a customer request, not specifically, but asking to make the check more performant. But you are correct, it's only holding the mutex once per check run.